Cloud Application Manager/Dedicated Edition/camd-vsphere.md

```diff
@@ -19,78 +18,68 @@
 * [Next Steps](#next-steps)
 * [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
 
-
 ### Overview
 
-
-This article explains that Cloud Application Manager Data Center Edition (appliance) can run as a virtual appliance in your vCenter private cloud. Here are the requirements, high-level steps to install appliance.
-
+This article explains that Cloud Application Manager Dedicated Edition (appliance) can run as a virtual appliance in your vCenter private cloud. Here are the requirements, high-level steps to install the appliance.
```

Comment:
I tweaked the second one a bit, but without simplifying as much as you suggested, since it's better to repeat more the keywords (such as 'requirements') to rank the page higher in search results.